docs/pages/machine-id/getting-started.mdx

```diff
@@ -157,8 +161,27 @@ the foreground to better understand how it works.
      --ca-pin=sha256:1111111111111111111111111111111111111111111111111111111111111111 \
```

Comment:
Don't we have some sort of variable we can use for tokens and CA pins in the docs?